cpp/example/test.c

```diff
@@ -85,8 +79,32 @@ int main(){
    b4.data = (unsigned char*)malloc(37*sizeof(unsigned char));
    
    printf("storage size of st2 is %d \n",storage_size(st2));
+
+   void* subrange = subrange_new(st2, 0 , UINT64_MAX);
+   if (subrange == NULL){
+      perror("failed to init subrange");
+   }
+   printf("subrange init successful with size %d \n ", subrange_size(subrange) );
+
+   void* subrange1 = subrange_new(st1, 0 , UINT64_MAX);
+   if (subrange == NULL){
+      perror("failed to init subrange");
+   }
+   printf("subrange init successful with size %d \n ", subrange_size(subrange1) );
+
+   void* ngn_inst1 = negentropy_new(subrange1, 153600);
```

Comment:
If possible, I'd use `define` statement instead of hard-coded value. In this case, maybe `#define MAX_MSG_SIZE 153600`